File: 251005/N번 더하기/add-n-times.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a, b, c;
    cin >> a >> b;
    c = a+b;
    for(int i; i < b; i++) {
        cout << c << endl;
        c += b;
    }
    return 0;
}

File: 251027/N번 더하기/add-n-times.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        string[] input = Console.ReadLine().Split();
        int a = int.Parse(input[0]);
        int b = int.Parse(input[1]);

        for(int i = 1; i <= b; i++) {
            a += b;
            Console.WriteLine(a);
        }
    }
}
